### PR TITLE
Feature/VTN-20276 Multiple engine picker and selection info panel

### PR DESCRIPTION
**Pull request by atungw**

### CODE REVIEW

File: packages/veritone-widgets/src/redux/modules/multipleEngineSelection/index.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        },
+        count
+      },
+      me {
+        organizationId
+      }
+    }
+  `;
+
+  return await callGraphQLApi({
```
Comment (gitrite) [inline packages/veritone-widgets/src/redux/modules/multipleEngineSelection/index.js]: not a real issue but you can omit `await` if you're returning from an asynchronous operation inside an async function

File: packages/veritone-widgets/src/widgets/EnginePicker/MultiplePicker.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  static propTypes = {
+    canSelectMore: bool,
+    selected: objectOf(any),
+    toggleSelection: func
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.id = guid();
```
Comment (gitrite) [inline packages/veritone-widgets/src/widgets/EnginePicker/MultiplePicker.js]: i'm thinking more descriptive property names would help here? just to make the code more self-documenting

File: packages/veritone-widgets/src/widgets/EnginePicker/SelectionInfoPanel.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    return (
+      <Paper elevation={4} style={{ height: '100%', minHeight: '300px' }}>
+        <div className={styles.pickerTitle}>
+          <Typography variant="headline">
+            Selected Engines{' '}
+            <Typography
+              variant="subheading"
+              style={{ display: 'inline-block' }}
+            >
+              {Object.keys(this.props.selected).length} / 6
```
Comment (gitrite) [inline packages/veritone-widgets/src/widgets/EnginePicker/SelectionInfoPanel.js]: hard coded values scare me. can we put it in a variable that helps to make it clear what it's for? additionally, is there value in making it a prop?

Comment (atungw) [reply packages/veritone-widgets/src/widgets/EnginePicker/SelectionInfoPanel.js]: Oops, yeah I meant to make that a prop.

**Pull request merged**